torchao/quantization/quant_primitives.py

```diff
@@ -2185,8 +2185,9 @@ def _choose_scale_float8(
     block_size: List[int],
     float8_dtype: torch.dtype = torch.float8_e4m3fn,
     scale_dtype: torch.dtype = torch.float32,
-    hp_value_lb: Optional[float] = None,
+    hp_value_lb: Optional[float] = 1e-12,
     hp_value_ub: Optional[float] = None,
+    reverse: bool = False,
```

Comment:
I think we should make the callsites match instead of having a flag

Reply:
yeah this is not the final state, was just trying out things to check if this helps or not

Reply:
Edit: missed this one during last update, but removed locally, please review the rest of the changes